bin/main.dart

```diff
@@ -6,8 +6,14 @@ import 'package:args/args.dart';
 void main(List<String> args) {
   final parser = ArgParser(allowTrailingOptions: true);
   parser.addMultiOption('localization', abbr: 'l', help: 'Localization codes');
+  parser.addOption('dirPath', abbr: 'd', defaultsTo: 'assets/i18n', help:'i18n files directory, \'assets/i18n\' by default');
   final argResults = parser.parse(args)['localization'];
-  final dirPath = 'assets/i18n';
+  if( argResults.isEmpty ) {
```

Comment:
`if(argResults.isEmpty) {`

without spaces. Also cruicial for the pub https://pub.dev/packages/localized.
Please check out Dart code guiding https://dart.dev/guides/language/effective-dart/style